lib/point_quest_web/live/quest.ex

```diff
@@ -20,15 +21,21 @@ defmodule PointQuestWeb.QuestLive do
         actor={@actor}
         quest_objective={@quest_objective}
       />
-      <div class={"#{if @round_active?, do: "visible", else: "invisible"} flex gap-x-2 items-center"}>
-        <.icon name="hero-arrow-top-right-on-square" />
-        <%= if @quest_objective != "" do %>
-          <a href={@quest_objective} target="_blank" class="text-indigo-500">
-            <%= @quest_objective %>
-          </a>
-        <% else %>
-          N/A
-        <% end %>
+      <div class="flex justify-between">
+        <div class={"#{if @round_active?, do: "visible", else: "invisible"} flex gap-x-2 items-center"}>
```

Comment:
Adjusted structure so that audio button is to the far right of the quest objective.

![image](https://github.com/OSFFriends/PointQuest/assets/13260877/dbae876b-273e-4d83-a60f-12924f36202d)